client/devdocs/docs-selectors/result.jsx

```diff
@@ -28,27 +32,33 @@ export default function DocsSelectorsResult( { url, name, description, tags, exp
 				{ description || <em>No description available</em> }
 			</p>
 			<div className="docs-selectors__result-io">
-				{ paramTags.length > 0 && (
+				{ params.length > 0 && (
 					<div className="docs-selectors__result-arguments">
 						<span className="docs-selectors__result-label">Arguments</span>
-						{ paramTags.map( ( tag ) => (
-							<div className="docs-selectors__result-arguments-content" key={ tag.name }>
-								<div className="docs-selectors__result-arguments-name">
-									<strong>{ tag.name }</strong>
-									<DocsSelectorsParamType { ...tag.type } />
+						{ params.map( param => (
+							<div key={ param.name }>
+								<div className="docs-selectors__result-arguments-content">
+									<strong className="docs-selectors__result-arguments-name">
+										{ param.variable && '...' }
+										{ param.optional && '[' }
+										{ param.name }
+										{ param.hasOwnProperty( 'defaultvalue' ) && ' = ' + param.defaultvalue }
```

Comment:
Optional (and maybe better to standardize on `hasOwnProperty` given the gotchas of `in`) but this could be shortened to...

```js
'defaultvalue' in param
```

...without any negative consequences.